modules/manage/pages/security/fips-compliance.adoc

```diff
@@ -0,0 +1,63 @@
+= Configure Redpanda for FIPS 
+:description: Configure Redpanda to operate in FIPS compliance mode.
+:page-aliases: security:fips-compliance.adoc
+
+Redpanda platform (Redpanda Enterprise) provides FIPS-compliant cipher enforcement for brokers using OpenSSL 3.0.9, which is https://csrc.nist.gov/projects/cryptographic-module-validation-program/certificate/4282[validated^] for https://csrc.nist.gov/pubs/fips/140-2/upd2/final[140-2^] and is undergoing validation by NIST for https://csrc.nist.gov/pubs/fips/140-3/final[140-3^]. Both Redpanda and `rpk` leverage validated OpenSSL libraries for all security-related cryptography operations.
```

Comment:
@deniscoady: This wording (Redpanda Enterprise) makes is sound like it requires an Enterprise license. Is that correct? Do we need to update this [list of enterprise features](https://deploy-preview-562--redpanda-docs-preview.netlify.app/24.2/get-started/licenses/#redpanda-enterprise-edition) to add FIPS compliance?

Reply:
Yes this does require an enterprise license

Reply:
thanks, I've added the [admonition](https://deploy-preview-562--redpanda-docs-preview.netlify.app/24.2/manage/security/fips-compliance/) we include on enterprise features & added it to the [list of enterprise features](https://deploy-preview-562--redpanda-docs-preview.netlify.app/24.2/get-started/licenses/#redpanda-enterprise-edition).